resources/measures/nighttime_oa_controls/tests/new_measure_test.rb

```diff
@@ -0,0 +1,323 @@
+
+    osm_path = model_input_path(osm_name)
+	osm_path_output = model_output_path(osm_name)
+    epw_path = epw_input_path(epw_name)
+
+    # Create an instance of the measure
+    measure = NighttimeOAControls.new
+
+    # Load the model; only used here for populating arguments
+    model = load_model(osm_path)
+    arguments = measure.arguments(model)
+    argument_map = OpenStudio::Measure::OSArgumentMap.new
+	
+	# Apply the measure to the model and optionally run the model
+    result = apply_measure_and_run(__method__, measure, argument_map, osm_path, epw_path, run_model: true)
+	#result = apply_measure_and_run(__method__, measure, osm_path, epw_path, run_model: true)
+    assert_equal('Success', result.value.valueName)
+	
+    template = 'ComStock 90.1-2013'
+    std = Standard.build(template)
+	
+	#load model with measure applied 
+	#model = load_model(osm_path_output)
+	
+    # loop through air loops
+    unitary_system_count = 0
+    li_unitary_systems = []
+    non_unitary_system_count = 0
+    li_non_unitary_systems = []
+	print('test') 
+    model.getAirLoopHVACs.sort.each do |air_loop_hvac|
+	  puts("in loop")
+      # skip systems that are residential, use evap coolers, or are DOAS
+      next if NighttimeOAControls.air_loop_res?(air_loop_hvac)
+      next if NighttimeOAControls.air_loop_evaporative_cooler?(air_loop_hvac)
+      next if NighttimeOAControls.air_loop_doas?(air_loop_hvac)
+      # skip data centers
+      next if ['Data Center', 'DataCenter', 'data center', 'datacenter', 'DATACENTER', 'DATA CENTER'].any? { |word| (air_loop_hvac.name.get).include?(word) }
+      # check unitary systems
+      if NighttimeOAControls.air_loop_hvac_unitary_system?(air_loop_hvac)
+        unitary_system_count += 1
+        li_unitary_systems << air_loop_hvac
+      else
+        li_non_unitary_systems << air_loop_hvac
+        non_unitary_system_count += 1
+      end
+    end
+	
+    li_non_unitary_systems.sort.each do |air_loop_hvac|
+
+      # change night OA schedule to match hvac operation schedule for no night OA
+	  #look at schedule in base case 
+      #case rtu_night_mode
+      #when 'night_fancycle_novent'
+        # Schedule to control whether or not unit ventilates at night - clone hvac availability schedule
+       if air_loop_hvac.airLoopHVACOutdoorAirSystem.is_initialized
+             air_loop_oa_system = air_loop_hvac.airLoopHVACOutdoorAirSystem.get.getControllerOutdoorAir
+             min_oa_sched = air_loop_oa_system.minimumOutdoorAirSchedule.get()
+			 if min_oa_sched.to_ScheduleRuleset.is_initialized
+			    puts('sched loop')
+			    min_oa_sched = min_oa_sched.to_ScheduleRuleset.get
+			    puts(min_oa_sched)
+				#assert(true) 
+			end 
+			if min_oa_sched.to_ScheduleConstant.is_initialized
+			    puts('sched loop')
+			    min_oa_sched = min_oa_sched.to_ScheduleConstant.get
+			    puts(min_oa_sched)
+				#assert(false) 
+			end 
+       end 
+	  
+	  end 
+
+    end
```

Comment:
remove extra space and indentations

Reply:
I see an assertion on line 147 for the result of the model, but I do not see any assertions below that. The intent should be to assert that certain conditions are met between the initial and final model state. For example, in the final model, you could assert the OA schedule matches your expectation.